package.json

```diff
@@ -20,6 +20,7 @@
     "test": "cross-env NODE_ENV=test NODE_TLS_REJECT_UNAUTHORIZED=0 mocha -t 16000 --require babel-register test/**/*.js",
     "test:unit": "cross-env NODE_ENV=test mocha --require babel-register test/unit/**/*.js",
     "test:int": "cross-env NODE_TLS_REJECT_UNAUTHORIZED=0 mocha -t 4000 --require babel-register test/integration/**/*.js",
+    "test:replica:set": "./test/resources/replica-set-test/setup.sh",
```

Comment:
Lets add a cleanup command here to make it easier to clean up after the replica set test.